extensions/amp-access-poool/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'dackmin'}, {name: 'dvoytenko'}, {name: 'jpettitt'}],
```

Comment:
Should we use wg- group as an owner here?

Reply:
I don't think so because as we add wg members from different companies I don't think they should own each others code.

Reply:
makes sense. thanks!